src/V3Ast.h

```diff
@@ -31,6 +31,7 @@
 #include "V3Ast__gen_forward_class_decls.h"  // From ./astgen
 
 #include <cmath>
+#include <cstdio>
```

Comment:
What needs this?

Reply:
Nothing, I added it by mistake - thanks for pointing this out. It's gone now.